flit.ini

```diff
@@ -3,8 +3,8 @@ module=gidler
 author=Caleb Hattingh
 author-email=caleb.hattingh@gmail.com
 home-page=https://github.com/cjrh/google-images-downloader
-requires=python-slugify (>=1.2)
-    git+https://github.com/cjrh/chromote.git@switch-to-ws4py
+requires=unicode-slugify (>=0.1.3)
+    https://github.com/mozilla/unicode-slugify.git
```

Comment:
This must remain as `git+https://github.com/cjrh/chromote.git@switch-to-ws4py`.  It is the other dependency (chromote). Only the prior line, must change, i.e., the one you changed to `unicode-slugify`.